src/policy/CMSRucioPolicy/algorithms/lfn2pfn.py

```diff
@@ -0,0 +1,199 @@
+#! /usr/bin/env python3
```

Comment:
This file is just a copy of something we already have, right? Can you use git mv so that we keep the history of that file?

Reply:
Used `git mv` on relevant files. 
Later modified them as needed. 

Thankyou for the suggestion.